ntbs-service/Pages/ContactDetails/CaseManager.cshtml

```diff
@@ -0,0 +1,10 @@
+﻿@page "/ContactDetails/CaseManager/{Username}"
+@model ntbs_service.Pages.ContactDetails.CaseManagerModel
+
+@{
+    Layout = "Shared/_Layout";
+    ViewData["Title"] = $"Case Manager details";
```

Comment:
Same here - perhaps `"User contact details"`. Or, if we can get it easily (which we should be able to, as it's part of the url): `"{Username} contact details"`?